custom_components/waste_collection_schedule/waste_collection_schedule/source/abfall_neunkirchen_siegerland_de.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import requests
+from waste_collection_schedule import Collection
+from waste_collection_schedule.service.ICS import ICS
+
+TITLE = "Abfall Neunkirchen Siegerland"
+DESCRIPTION = " Source for 'Abfallkalender Neunkirchen Siegerland'."
+URL = "https://www.neunkirchen-siegerland.de"
+TEST_CASES = {
+    "Waldstraße":{ "street":"Waldstr"}
+}
+
+_LOGGER = logging.getLogger(__name__)
+
+class Source:
+    def __init__(self, searchterm):
```

Comment:
Please change `searchterm` also to `strasse`. Otherwise it doesn't match to the documentation.

Reply:
@mampfes now it is changed. Sorry for that. In my dev its ok, but didn't commited it.